week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,21 @@
+
+let getData = document.getElementById("getData");
+getData.addEventListener("click",()=>{
+    fetch("https://dog.ceo/api/breeds/image/random")
+    .then(Response => {
+        return Response.json();
+    })
+    .then(data =>{
+        console.log(data);
+        let listShow = document.getElementById("listShow");
+        let listEle = document.createElement("li");
+        let img = document.createElement("img");
+        listShow.appendChild(listEle);
+        listEle.appendChild(img);
+        img.src = data.message;
+    })
+    .catch(error =>{
+        console.log(error);
```

Comment:
Note that this is only visible to developers, most people don't look at the console - when handling errors, think if there's any information the end user would need to understand what happened

Reply:
I think it's good to use of 'console.error(error)'